support/index.html

```diff
@@ -166,6 +166,30 @@ <h2>Experienced Providers</h2>
         Through these relationships experienced providers are better able to provide timely and
         informed service.
     </p>
+    <div class="row tbl">
+         <div class="col-xs-2 cell text-center">
+            <img src="img/afrigis-logo.jpg"></img>
+        </div>
+        <div class="col-xs-10"><p>
+            <a href="https://www.afrigis.co.za/">AfriGIS</a> (South Africa) has been providing geospatial data in Africa since 1997. We specialise in international standards compliant location-sensitive data and solutions. We provide web tools and <a href="https://developers.afrigis.co.za">APIs</a> to help customers connect to, enhance and enrich their data with location information.
+</p><p>
+			AfriGIS operates and manages many GeoServer instances in our own and client cloud environments. We also provide custom development, training and support for GeoServer and QGIS. We believe that big businesses should help open-source projects in any way they can. This includes providing funding, writing documentation and custom plugins, offering support, and fixing issues.
+</p><p>
+			Visit our website for more of our geospatial data technology or <a href="https://www.afrigis.co.za/contact-us/">contact us</a> for support.
+        </p></div>
+        <div class="col-xs-10">
+            <ul class="experience">
+               <li>training</li>
+               <li>support</li>
+               <li>development</li>
+               <li>documentation</li>
+               <li>committer</li>
+               <li>service</li>
+               <li>product</li>
```

Comment:
What is your product that uses geoserver?

Reply:
Sorry, internally we call it a product, but it is actually more of a service.  I will remove.